add writers-program and cfp-program pages with revision (#85)
metalbear-co/metalbear.co: content/en/pages/cfp-program/index.md

---
title: CFP Program
description: Talk about mirrord
slug: cfp-program

> header_title: Talk About Mirrord

Reviewer: ```suggestion
header_title: Talk About mirrord
```

header_text: >
  Are you interested in speaking about mirrord at conferences?

>   We are calling on mirrored users, contributors, developers, and engineers actively working with mirrord to represent mirrord at conferences worldwide. Got a presentation in mind? This is your chance to share your knowledge and help spread the word about mirrord.

Reviewer: ```suggestion
  We are calling on mirrord users and contributors actively working with mirrord to talk about their takes and experiences with mirrord at conferences worldwide. Got a presentation in mind? This is your chance to share your knowledge and help spread the word about mirrord.
```

>   We are calling on mirrord users and contributors actively working with mirrord to talk about their takes and experiences with mirrord at conferences worldwide. Got a presentation in mind? This is your chance to share your knowledge and help spread the word about mirrord.

Reviewer: ```suggestion
  We're inviting mirrord users to talk about their takes and experiences with mirrord at conferences worldwide. Have an idea for a mirrord-related conference talk or workshop? This is your chance to share your knowledge and help spread the word about mirrord.
```

textbox:
- label: info
  title: Program details
  content: >
    - We offer financial support to cover the costs of travel, accommodation, and registration fees.
  
    - We're especially interested in supporting talks about mirrord, its use cases, or its functionalities.
    

>     - We will provide personalized assistance to help you deliver smooth presentations

Reviewer: ```suggestion
    - We will provide personalized assistance to help you deliver a great presentation.
```

> - label: needs

Reviewer: ```suggestion
- label: requirements
```

  title: What are we looking for?
  content: >

>     - Subject matter experts: users and individuals with deep knowledge and proven experience with mirrord and its features.

Reviewer: ```suggestion
    - Subject matter experts: users and individuals with deep knowledge and experience with mirrord.
```

>     - mirrord enthusiasts: Passionate about mirrord's potential and eager to share its benefits with the world.

Reviewer: ```suggestion
    - mirrord enthusiasts: people who are passionate about mirrord's potential and are eager to share its benefits with the world.
```

>     - Engaging presenters: Ability to deliver captivating presentations that inspire and educate audiences.

Reviewer: ```suggestion
    - Engaging speakers: speakers who can deliver captivating talks that inspire and educate technical audiences.
```

>     - Individuals from diverse backgrounds and fields are encouraged to apply as long as your topic intersects with the mirrord capabilities.

Reviewer: ```suggestion
    - Individuals from diverse backgrounds and fields are encouraged to apply as long as your topic intersects with mirrord.
```

how_to_apply:

>   - "Pick your conferences: Indicate the specific conferences you plan to attend and present at."

Reviewer: The first letter after a ":" doesn't need to be capitalized - please fix where relevant.

Reviewer 2: Fixed

>   - "Tell us about your talk: Briefly describe your proposed topic, the target audience, how it aligns with the themes of the conference, and demonstrates mirrord capabilities."

Reviewer: ```suggestion
  - "Tell us about your talk: Briefly describe your proposed topic, the target audience, how it aligns with the themes of the conference, and demonstrates mirrord's capabilities and use cases."
```
